### DISCUSSION TITLE
IS_DARK missing in v 4.3.4

### DISCUSSION THREAD

**Discussion by raubv0gel** [chillerlan/php-qrcode#158] [General]
Can you please release a new version? `IS_DARK` missing in v4.3.4.

**Comment by codemasher** [comment]
Soon:tm:

Jokes aside - I'm waiting for the official relase of PHP 8.2 to see if there are any unexpected errors happening - in the meantime you could just use `dev-main#c03aafcf183e297128931c40fb425fb40057ea86`. The encoder API probably won't change; there are some things in the QR Code reader that require some fixes though.

**Reply by raubv0gel** [reply]
Thank you!

Just right after writing the „issue“ I placed … `"chillerlan/php-qrcode": "dev-main#c03aafcf183e297128931c40fb425fb40057ea86",` 😄

**Comment by kasperkamperman** [comment]
PHP 8.2 was released on the 8th of December. Could the main branch (and composer version) be updated? I'm running in a lot of examples that are broken because of IS_DARK missing. 

Thanks in advance for the effort.

**Reply by codemasher** [reply]
Hi, I still have several things to do in the dev branch until it is ready to release. In the meantime you could just [install from `dev-main` with a commit tag](https://github.com/chillerlan/php-qrcode/wiki/Installation) (i know that's not necessarily convenient).
